app/src/main/java/be/huffle/allowanceplanner/ui/allowance/AllowanceFragment.java

```diff
@@ -14,18 +17,61 @@
 
 import be.huffle.allowanceplanner.R;
 
+import static android.content.Context.MODE_PRIVATE;
+
 public class AllowanceFragment extends Fragment
 {
-
 	private AllowanceViewModel allowanceViewModel;
+	private SharedPreferences sharedPreferences;
+	private Editor editor;
```

Comment:
Does it make sense to make these two properties instead of just local variables? They seem to only be used within a single function.